cmd/snap-update-ns/utils.go

```diff
@@ -132,105 +133,89 @@ func (sec *Secure) OpenPath(path string) (int, error) {
 	return fd, nil
 }
 
-// MkPrefix creates directories for all but the last segment and returns the
-// file descriptor to the leaf directory. This function is a base for secure
-// variants of mkdir, touch and symlink. None of the traversed directories
-// can be symbolic links.
-func (sec *Secure) MkPrefix(segments []string, perm os.FileMode, uid sys.UserID, gid sys.GroupID) (int, error) {
-	logger.Debugf("secure-mk-prefix %q %v %d %d -> ...", segments, perm, uid, gid)
-
-	// Declare var and don't assign-declare below to ensure we don't swallow
-	// any errors by mistake.
-	var err error
-	var fd int
+// MkPrefix creates all the missing directories in a given base path and
+//
```

Comment:
Empty line, sentence continues just below.

Reply:
Thanks for spotting this, I'll remove the newline.